### ISSUE TITLE
Is there a way to toggle draggable status without unmout/mount element

### ISSUE THREAD

**Issue by m7luffy** [STRML/react-grid-layout#756]
Thanks for submitting an issue to RGL!

Please mark the type of this issue:

- [ ] Bug
- [ ] Feature Request
- [ √] Question

If you have a question or bug report, please use the [WebpackBin Template](https://www.webpackbin.com/bins/-Kvr2qCxorvGMgVMxkmI)
to demonstrate. It is much easier for us to help you if you do.

I need a function to toggle draggable status. I do this by changing 'static' or 'isDraggable' property in layout. But changing these two properties will lead element unmounted. Mount a new element take a while in my app because it require data from network. Is there any way to change the status without re-mount the element?

**Comment by luminaxster** [comment]
**TL;DR:** Try providing your own  ```draggableHandle=".className"```:

Example:

``` <Responsive draggableHandle=".draggable" > 
<div key="x" >
            <div className={isDraggable?'draggable': 'disabled'}> ...</div>
</div>
...
</Responsive>
```

**Comment by m7luffy** [comment]
Thanks @luminaxster , this way does work.

**Comment by trydzynski-zen** [comment]
I'm impacted by this as well. When I toggle isDraggable and isResizable, all widgets are recreated.
My widgets (the draggable component) cache their queries, so remounting them does not cause network load, however they blink and/or animate which looks poorly.

Could this be fixed in the library, e.g. by calling functions, instead of adding Resizable/Draggable mixins via wrapping in new components? That should be enough to unlock react's reuse of components (I have verified the cause, but not the fix).
